plone/app/workflow/browser/sharing.py

```diff
@@ -50,7 +48,6 @@ class SharingView(BrowserView):
     # Actions
 
     template = ViewPageTemplateFile("sharing.pt")
```

Comment:
This should go as well, right?

Reply:
yes, wasn't done yet, hence the draft status ;)